src/helpers/value-from-css-variable.ts

```diff
@@ -8,7 +8,16 @@ import { CssVariable } from './types';
  */
 export function valueFromCssVariable(
   cssVariable: CssVariable,
-  target = document.documentElement,
+  target?: HTMLElement,
 ) {
-  return getComputedStyle(target).getPropertyValue(cssVariable) || null;
+  // If document isn't valid, then we cannot get the value of the CSS variable
+  if (typeof document === 'undefined') {
+    return null;
+  }
+
+  return (
+    getComputedStyle(target || document.documentElement).getPropertyValue(
```

Comment:
Since it’s a `null`/`undefined` check, the following feels a bit more precise:
```suggestion
    getComputedStyle(target ?? document.documentElement).getPropertyValue(
```
(and will be enforced by the DataCamp ESLint preset)